src/client/src/app/lib/components/feedback-banner.tsx

```diff
@@ -0,0 +1,22 @@
+import Tag from '@components/nhsuk-frontend/tag';
+import '../../global.css';
```

Comment:
I don't think you need this as `global.css` is already imported through `layout.tsx`